build.sc

```
@@ -653,7 +655,7 @@ trait Cli3 extends Cli {
       mainArgs = Seq(cache.toNIO.toString, classpath),
       workingDir = os.pwd
     )
-    val cp = res.out.text
+    val cp = res.out.lines.head
```

Are you sure lines will contain at least one element? maybe it would be better to throw a descriptive error

---

I've added a comment to address your point

---

Wouldn't calling `.trim` on `res.out.text` just work?